src/main/kotlin/com/workos/users/UsersApi.kt

```diff
@@ -761,4 +763,63 @@ class UsersApi(private val workos: WorkOS) {
   fun listAuthFactors(id: String): AuthenticationFactorList {
     return workos.get("/users/$id/auth/factors", AuthenticationFactorList::class.java)
   }
+
+  /**
+   * Parameters for the [enrollAuthFactor] method.
+   */
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  class EnrollAuthFactorOptions @JvmOverloads constructor(
+    @JsonIgnore
```

Comment:
Don't have too much context on kotlin libraries, but I'm assuming this is to obfuscate the `userId` from the serialized json?